scripts/stats.js

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env node
+
+'use strict';
+
+var _ = require('lodash');
+var mongodb = require('mongodb');
+var moment = require('moment');
+var async = require('async');
+var config = require('../config');
+
+
+var c = config.storageOpts.mongoDb;
+var url = 'mongodb://' + (c.host || 'localhost') + ':' + (c.port ||  27017) + '/bws';
+
+var startDate = moment();
+var endDate = moment();
+
+var stats = {};
+
+var wallets = {};
+
+var bwsStats = {};
+
+bwsStats.cleanUp = function() {
+  stats = {
+    'livenet': {},
+    'testnet': {}
+  };
+}
+
+
+bwsStats.AddingWalletToCache = function(data) {
+  if (!data) return;
+  wallets[data.id] = data.network;
+};
+
+bwsStats.TotalNewWalletForToday = function(data) {
+  if (!data) return;
+  var day = moment(data.createdOn * 1000).format('YYYYMMDD');
+  if (!stats[data.network][day]) {
+    stats[data.network][day] = {
+      totalTx: 0,
+      totalAmount: 0,
+      totalNewWallets: 0
+    };
+  }
+  stats[data.network][day].totalNewWallets++;
+};
+
+bwsStats.TotalTxpForToday = function(data) {
```

Comment:
The name of this function is misleading. It computes txp stats for any day in the interval and not just for the current day.